src/components/Navigation.js

```diff
@@ -78,6 +79,7 @@ class Navigation extends React.Component {
                   <ConditionalNavigation to="/token_balances" label="Token balances" featureToggle={`${UNLEASH_TOKEN_BALANCES_FEATURE_FLAG}.rollout`} />
                 </div>
               </li>
+              <ConditionalNavigation to="/timeseries" label="Time&nbsp;Series" featureToggle={`${UNLEASH_TIME_SERIES_FEATURE_FLAG}.rollout`} />
```

Comment:
Why don't we use a single space instead of `&nbsp;`?

Reply:
Will remove this line on the next commit. But, when I was using the single space, the `Time Series` navigation link was being split in two lines.